ravenframework/Optimizers/Optimizer.py

```diff
@@ -175,13 +187,13 @@ def __init__(self):
     self._constraintFunctions = []      # list of constraint functions
     self._impConstraintFunctions = []   # list of implicit constraint functions
     self._requireSolnExport = True      # optimizers only produce result in solution export
-    self.optAssemblerList = ['DataObjects', 'Distributions', 'Functions', 'Files'] # List of assembler entities required to initialize an optmizer
+    self.optAssemblerList = ['DataObjects', 'Distributions', 'Functions', 'Files', 'Models'] # List of assembler entities required to initialize an optmizer
```

Comment:
See my previous comment, change 'Models' to 'ROM'

Reply:
Changing Models to ROM here results in an error when generating the assembler objects. I believe that this list can only consist of so called 'Main' nodes.